apis/proto/vald/insert.proto

```diff
@@ -0,0 +1,42 @@
+//
+// Copyright (C) 2019-2020 Vdaas.org Vald team ( kpango, rinx, kmrmt )
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+syntax = "proto3";
+
+package vald;
+
+option go_package = "github.com/vdaas/vald/apis/grpc/vald";
+option java_multiple_files = true;
+option java_package = "org.vdaas.vald";
+option java_outer_classname = "ValdApi";
```

Comment:
![badge](https://img.shields.io/badge/review-must-red.svg?style=flat-square)
Could you please modify the Java package and class name correctly?
I recommend to modify like this:
```suggestion
option java_package = "org.vdaas.vald.insert";
option java_outer_classname = "InsertApi";
```

Reply:
I agree with classname change but do we need change package name?

Reply:
It is okay not to change, but `org.vdaas.vald` is already used as root package in other protos. so it is better to make a new package like `org.vdaas.vald.xxx`.

Reply:
how about to use `org.vdaas.vald.vald`?

Reply:
How about `org.vdaas.vald.api` ?

Reply:
sounds good :+1: